react/features/visitors/middleware.ts

```diff
@@ -390,7 +416,12 @@ function _subscribeVisitorsList(getState: IStore['getState'], dispatch: IStore['
 
                 dispatch(updateVisitorsList(visitors));
             },
-            getState()['features/base/jwt'].jwt);
+            getState()['features/base/jwt'].jwt,
+            // Connect callback - called when WebSocket connection is established
+            () => {
+                logger.debug('Visitors list WebSocket connection established');
+                setupTimeout(); // Reset timeout when connection is established
```

Comment:
Those retries can go forever, right? I'm wondering if we should stop them after a few retries just to not flood the backend. Maybe if we have failed 3 times we wouldn't succeed on the 4th as well. WDYT?